static/_redirects

```diff
@@ -251,6 +251,7 @@
 /docs/tasks/configure-pod-container/weave-network-policy/     /docs/tasks/administer-cluster/weave-network-policy/ 301
 /docs/tasks/debug-application-cluster/sematext-logging-monitoring/     https://sematext.com/kubernetes/ 301
 /docs/tasks/job/work-queue-1/     /docs/concepts/workloads/controllers/jobs-run-to-completion/ 301
+/docs/tasks/setup-konnectivity/setup-konnectivity/   /docs/tasks/access-kubernetes-api/setup-konnectivity/ 301
```

Comment:
Wouldn't we also have to setup redirects for everything under `/docs/tasks/access-kubernetes-api/*` to go to `/docs/tasks/extend-kubernetes/*`?

Reply:
Right, this is why I am considering alternatives to renaming the directory.

Reply:
There are only ~7 files right now. We could do it even if means having to setup 7 redirects.

Reply:
Another way would to be retitle the `/docs/tasks/access-kubernetes-api/_index.md` to something like `access-api` or `using-api` (reference section has this title) and then create a new directory called `extend-kubernetes` to host the other task pages.
That way the "api" related tasks are left as is and any other tasks can be moved into the
new directory. Thoughts?

Reply:
Most of the pages under `/docs/tasks/access-kubernetes-api` seem to be about extending kubernetes rather than accessing api. So I am in favor of just renaming the directory and creating the necessary redirects.

Reply:
I am hoping to keep the API-related tasks separate from the other "extension" tasks.

Reply:
@prasadkatti , Let me think on this a bit more. I want to make sure the new directory locations will cover any other "extension" tasks as well.

Reply:
@prasadkatti, I compared the current task and concept directory structures.
The contents under `access-kubernetes-api` contain a mix of tasks related to extending the API (and API server) as well as setting up an HTTP proxy to access the API.
The corresponding `concept` content is found under `/docs/concepts/extend-kubernetes/api-extension`.
I am thinking about creating or moving the content under `access-kubernetes-api` to a subdirectory under the new `docs/tasks/extend-kubernetes/extend-api/` directory. The Konnectivity svc page would fall under extend-kubernetes as well as any scheduler, service catalog, proxy tasks.
What do you think about these possible changes?

Reply:
Just reiterated. If new locations look good, will update links and add redirects.